### ISSUE TITLE
Math rendering display mode

### ISSUE THREAD

**Issue by jaabio** [curiositry/mnml-ghost-theme#37]
Hi.  How do you handle display mode for math rendering with Katex?

**Comment by curiositry** [comment]
```
<section class="math">\Delta p \Delta x \geq {h \over 4 \pi}</section>
```

See: http://mnml.herokuapp.com/typesetting-mathematics-on-your-ghost-blog-with-tex-mnml-and-katex/

**Comment by jaabio** [comment]
Thanks.  So, I already tried some variation of that.  
But then I noticed that, in katex.hbs, `katex.render` is called without passing in the optional configuration parameter `{ displayMode: true }`.  
See [this](https://github.com/Khan/KaTeX#rendering-options) for behavior of `render`.  The gist of it is that the default value of `displayMode` is  `false`, which causes Katex to use math mode.  
I haven't dug deep into Katex or mnml, but is it possible that mnml is rendering in math mode, but displaying it on a separate line, such it looks like it's display mode, but it's not?
